include/class.filter.php

```diff
@@ -800,7 +800,6 @@ static function isAutoReply($headers) {
             'Precedence'        => array('AUTO_REPLY', 'BULK', 'JUNK', 'LIST'),
             'X-Precedence'      => array('AUTO_REPLY', 'BULK', 'JUNK', 'LIST'),
             'X-Autoreply'       => 'YES',
-            'X-Auto-Response-Suppress' => array('ALL', 'DR', 'RN', 'NRN', 'OOF', 'AutoReply'),
```

Comment:
After further review we agree but we will keep the `AutoReply` option to respect suppressing auto-replies:
```php
'X-Auto-Response-Suppress' => array('AutoReply'),
```

Reply:
Great, thanks for reviewing this